src/main/groovy/org/edx/jenkins/dsl/AnalyticsConstants.groovy

```diff
@@ -159,13 +159,15 @@ This text may reference other parameters in the task as shell variables, e.g.  $
         return {
             sshAgent('1')
             timestamps()
+            buildUserVars()
```

Comment:
Since I'm enabling this plugin in common_wrappers, it will effectively enable the below feature (to automatically notify the builder) on all manually built jobs, not just the event-export-incremental ones.  I can't think of any reason this would be bad, and the code seemed simpler this way.